plugins/@grouparoo/calculated-property/src/lib/test.ts

```diff
@@ -0,0 +1,5 @@
+import { TestPluginMethod } from "@grouparoo/core";
+
+export const test: TestPluginMethod = async () => {
+  return { success: true };
```

Comment:
This is probably always true.  Perhaps it might make sense to try out `vm2` here and run a `() => {return true}` or something like that?